### DISCUSSION TITLE
What are some of the more useful tricks for small targets?

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmdetection#8927] [General]
有哪些针对小目标比较有用的trick?
What are some of the more useful tricks for small targets?

**Comment by vansin** [comment]
1. Increasing your image capture resolution
2. Increasing your model's input resolution
3. Tiling your images
4. Generating more data via augmentation
5. Auto learning model anchors
6. Filtering out extraneous classes

**Comment by jokober** [comment]
I also plan to detect tiny objects in high resolution images. Currently I use darkhelp which supports image tiling but I would like to switch to mmdetection as I also want to use mmtracking.
